lib/Twig/Template.php

```diff
@@ -581,7 +581,7 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
                 return;
             }
 
-            throw new Twig_Error_Runtime(sprintf('Method "%s" for object "%s" does not exist', $item, get_class($object)), -1, $this->getTemplateName());
+            throw new Twig_Error_Runtime(sprintf('Neither the property "%1$s" nor one of the methods "%1$s()", "get%2$s()"/"is%2$s()" or "__call()" exist and have public access in class "%3$s"', $item, ucfirst($item), get_class($object)), -1, $this->getTemplateName());
```

Comment:
this will help you for the C extension btw, as you won't need to implement ucfirst anymore